index.js

```diff
@@ -237,11 +237,19 @@ RedisClient.prototype.create_stream = function () {
             this.stream.destroy();
         }
 
+        // Accept snake_case options and convert them to camelCase
+        var opts = {};
+        var keys = Object.keys(this.connection_options);
+        var key;
+        while (key = keys.pop()) {
+            opts[key.replace(/_([a-z])/g, function (g) { return g[1].toUpperCase(); })] = this.connection_options[key];
+        }
```

Comment:
The reason for the conversion is that node_redis was written in snake_case and camelCase support was added by converting these values to snake_case and therefore both is supported.